problemtools/verifyproblem.py

```diff
@@ -379,23 +389,23 @@ def check(self, args):
         return self._check_res
 
 
-    def run_submission(self, sub, args, timelim_low, timelim_high):
+    def run_submission(self, sub, args, timelim_low, timelim_high, subinfo):
```

Comment:
I am not happy about tacking on this subinfo argument down the callstack just to be able to prettyprint the error.

Remove this, and simply print `sub` for now (we lose the "ACR submission" part but that's OK for now).

Reply:
The subinfo argument is also used by #81 for more essential purposes, if that makes a difference. But I can remove it if we want to merge this early.

Reply:
I don't know how to parse your silence, but removed anyway.